src/pages/EnablePayments/AdditionalDetailsStep.js

```diff
@@ -73,25 +65,29 @@ const defaultProps = {
         idNumber: '',
         errorCode: '',
     },
-    walletAdditionalDetailsDraft: {
-        legalFirstName: '',
-        legalLastName: '',
-        addressStreet: '',
-        addressCity: '',
-        addressState: '',
-        addressZip: '',
-        phoneNumber: '',
-        dob: '',
-        ssn: '',
-    },
     ...withCurrentUserPersonalDetailsDefaultProps,
 };
 
+const INPUT_IDS = {
+    LEGAL_FIRST_NAME: 'legalFirstName',
+    LEGAL_LAST_NAME: 'legalLastName',
+    PHONE_NUMBER: 'phoneNumber',
+    DOB: 'dob',
+    SSN: 'ssn',
+    ADDRESS: {
+        street: 'addressStreet',
+        city: 'addressCity',
+        state: 'addressState',
+        zipCode: 'addressZip',
```

Comment:
Can we either capitalize these keys or turn them into one key like `ADDRESS_STREET`, `ADDRESS_CITY`, etc?

Reply:
Agree with capitalizing these if possible. I'd prefer that over `ADDRESS_STREET`, `ADDRESS_CITY`, etc...

Reply:
It looks like this got reverted to lowercase, mind fixing again? Sorry :D

Reply:
Oh looks like they can't be capitalized, due to some weirddddddddd stuff we're doing passing these INPUT_IDS into `AddressForm`, which passes them to `AddressSearch` - looks like we should put the input keys in CONST or something and just import them, not pass props around like that 🤷 Anyway, resolving